say_my_name("John", "Doe")

say_my_name("Alice")
try:
    say_my_name(123, "Smith")
except TypeError as e:
    print(f"Error: {e}")

try:
    say_my_name("Bob", 456)
except TypeError as e:
    print(f"Error: {e}")


say_my_name("", "")


try:
    say_my_name("", 789)
except TypeError as e:
    print(f"Error: {e}")
